public/index.css

```diff
@@ -161,20 +163,26 @@ button {
   margin-bottom: 1em;
 }
 #loading-indicator {
-  border: 5px solid rgb(73, 180, 87); 
-  border-top: 5px solid rgb(0, 0, 0); 
+  border: 5px solid transparent; 
```

Comment:
I made improvements to the styling of the the loading indicator and fixed the text spinning issue below.

Reply:
great